### PR TITLE
DC-274: Make Zip Import work again

### PR DESCRIPTION
**Pull request by patrick-schweizer**
Jira: https://issues.opennms.org/browse/DC-274

### CODE REVIEW

File: plugin/src/main/java/org/opennms/plugins/cloud/config/CertificateImporter.java [hunk 1]
```diff
@@ -84,26 +78,11 @@ public void doIt() throws IOException {
...
-        tryConnection();
     }
 
     public boolean isConfigStored(final Map<Type, String> config) {
         return Objects.equals(config.get(Type.privatekey), scv.getOrNull(Type.privatekey))
                 && Objects.equals(config.get(Type.publickey), scv.getOrNull(Type.publickey))
                 && Objects.equals(config.get(Type.tokenvalue), scv.getOrNull(Type.tokenvalue));
     }
-
-    void tryConnection() {
```
Comment (freddy61025) [inline plugin/src/main/java/org/opennms/plugins/cloud/config/CertificateImporter.java:97]: is that make a real connection to a server for testing? And it is not suppose to do that in test?

**Review by freddy61025 — APPROVED** [review]
LGTM only a question about the removed test

**Review by freddy61025 — COMMENTED** [review]
It seems got problem when install plugin.
image

File: plugin/src/main/java/org/opennms/plugins/cloud/config/CertificateImporter.java (hunk 1 above)
Comment (patrick-schweizer) [reply plugin/src/main/java/org/opennms/plugins/cloud/config/CertificateImporter.java:97]: Yes, it tries to call the health endpoint. I moved it to here:
https://github.com/OpenNMS/opennms-cloud-plugin/blob/jira/DC-274/plugin/src/main/java/org/opennms/plugins/cloud/config/ConfigurationManager.java#L131
=> we still have this check.

**Comment by patrick-schweizer** [comment]
> It seems got problem when install plugin. image
Hm it seems the health check on the cloud gateway is not implemented anymore... I will check with @stumac

File: plugin/src/main/java/org/opennms/plugins/cloud/config/ConfigurationManager.java [hunk 2]
```diff
@@ -221,6 +225,7 @@ public ConfigStatus configure() {
 
             initGrpcServices(cloudGatewayConfig); // give all grpc services the new config
             LOG.info("All services configured with grpc config.");
+            // checkConnection();
```
Comment (freddy61025) [inline plugin/src/main/java/org/opennms/plugins/cloud/config/ConfigurationManager.java:228]: you forgot remove comment

**Review by freddy61025 — DISMISSED** [review]

File: plugin/src/main/java/org/opennms/plugins/cloud/config/ConfigurationManager.java (hunk 2 above)
Comment (patrick-schweizer) [reply plugin/src/main/java/org/opennms/plugins/cloud/config/ConfigurationManager.java]: hups. fixed.

**Review by freddy61025 — APPROVED** [review]

**Pull request merged**